apps/react/src/features/anime/components/AnimeList/AnimeList.tsx

```diff
@@ -60,10 +69,15 @@ const AnimeListComponent: FC = () => {
   const dispatch = useAppDispatch();
   const animeList = useAppSelector(selectAnimeList);
   const isLoading = useAppSelector(selectIsAnimeLoading);
+  const params = useParams();
+  const navigate = useNavigate();
   const [searchParams, setSearchParams] = useSearchParams();
   const [queryParams, setQueryParams] = useState<AnimeListQueryParams>(
     getAnimeListParamsFromUrl(searchParams),
   );
+  const [currentAnimeId, setCurrentAnimeId] = useState<
+    AnimeDetail['id'] | null
+  >(Number(params['id']));
```

Comment:
I think it's more readable.
```suggestion
  const [currentAnimeId, setCurrentAnimeId] =
    useState<AnimeDetail['id'] | null>(Number(params['id']));
```